modules/reference/pages/xsd-schema.adoc

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: Open Liberty includes tools for creating an XSD schema for the `server.xml` file.
+:page-layout: general-reference
+:seo-title: XSD schema for the server.xml file
+:seo-description: Open Liberty includes tools for creating an XSD schema for the `server.xml` file.
+:page-type: general
+= XML schema for the server.xml file
+
+The XML Schema Definition (XSD) language is a markup language that defines the structure and data types of XML documents.
+Open Liberty includes tools that create an XSD file that generates a schema and validates the `server.xml` file of your application.
+
+== Create XML schema for Open Liberty with the Maven plug-in
+
+For https://maven.apache.org/what-is-maven.html[Maven] based projects with Open Liberty, run the `mvn package` command from the start directory to create the `./target/liberty/wlp/bin/tools` directory that contains the `ws-schemagen.jar` file.
+Create the XML schema by running the following command in the same directory where the `wlp` directory exists:
+
+[source,xml]
+----
+java -jar ./wlp/bin/tools/ws-schemagen.jar ./src/main/liberty/config/server.xsd
+----
+
+Set the location of the created XML schema to the same directory that the `server.xml` file exists.
+The default location for the  Open Liberty `server.xml` file is the `src/main/liberty/config` directory.
+
+The command creates XML schema `server.xsd` file in the location that is set in the command.
+
+== Create XML schema for Open Liberty
+
+Download the Open Liberty files and extract the archive, which can be found in the `wlp` directory.
+Create the XML schema by running the following command in the same directory where the `wlp` directory exists:
+
+[source,xml]
+----
+java -jar ./wlp/bin/tools/ws-schemagen.jar server.xsd
+----
+
+The command creates the XML schema `server.xsd` file in the location that is set in the command.
+You can now validate the structure of your `server.xml` file in any IDE of your choice.
+
+
+== Add schema location to the server.xml file
+
+After you create the XSD schema, add the schema location to the `xsi : noNamespaceSchemaLocation` attribute in the server element of the `server.xml` file as shown in the following snippet:
+
+[source,xml]
+----
+<server
+ description="my_server"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:noNamespaceSchemaLocation="server.xsd">
+----
+
```

Comment:
add clarification:

The schema location is set to `server.xsd` because the example command saved it in the same directory as the server.xml file. If you specify a different path for the output file,  adjust the noNamespaceSchemaLocation attribute accordingly.